include/klee/Internal/Module/KInstruction.h

```diff
@@ -39,7 +43,15 @@ namespace klee {
     unsigned dest;
 
   public:
-    virtual ~KInstruction(); 
+    virtual ~KInstruction();
+
+    void printFileLine(llvm::raw_ostream &debugFile) {
+      if (info->file != "")
+	debugFile << info->file << ":" << info->line;
+      else
+	debugFile << "[no debug info]";
+    };
+
```

Comment:
@ccadar I wonder if we really want to put the implementation in the header file. I would move it in the cpp file.